### PR TITLE
HW10: Seventeen 1 and 2

### PR DESCRIPTION
**Pull request by yestocode**

### CODE REVIEW

File: seventeen1.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import random
+
+
+def human():
+	total_marbles = 17
+	while (total_marbles > 0):
+		try:
+			human_marbles =int(raw_input("How many marbles do you want to remove (1-3) ?"))
+
+		except ValueError:
```
Comment (danielsgriffin) [inline seventeen1.py]: Good job!

File: seventeen1.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+					print "You removed {0} marbles.\n Number of marbles left in jar : {1}".format(human_marbles,total_marbles)
+					if total_marbles == 0:
+						print "There are no marbles left. Computer wins."
+						break
+					if human_marbles <= total_marbles:
+						computer_marbles = human_marbles
+					else:
+						computer_marbles = random.randint(1,total_marbles)
+					total_marbles = total_marbles - computer_marbles
+					print "Computer's turn...\n Computer removed {0} marbles.\n Number of marbles left in jar : {1}".format(computer_marbles,total_marbles)
```
Comment (danielsgriffin) [inline seventeen1.py]: watch your line length

File: seventeen1.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+					if total_marbles == 0:
+						print "There are no marbles left. You win."
+						break
+			
+def main():
+	human()
+
+main()
```
Comment (danielsgriffin) [inline seventeen1.py]: Good job.

more functions may be a good idea
try to have all the outputs as in the example (far easier for both you and me to check for differences)

File: seventeen2.py [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+	final_list = []
+	with open("output.txt","w") as fout:
+		with open("i206_placein_input_0.txt","r") as fin:
+			for line in fin:
+				
+				final_list=human(line.strip())
+				fout.write("Game #"+str(game_number)+". Play Sequence:"+final_list[0][1:]+". Winner:"+final_list[1]+"\n")
+				game_number+= 1
+
+
```
Comment (danielsgriffin) [inline seventeen2.py]: You forgot to write this line: 
"Player 1 won 5 times; Player 2 won 5 times."

**Comment by danielsgriffin** [comment]
Good job.

File: seventeen2.py (hunk 4 above)
Comment (yestocode) [inline seventeen2.py]: oh yea sry! will include it ! thanks

**Pull request closed without merging**